wayfire/ipc.py

```diff
@@ -10,33 +10,42 @@ class WayfireSocket:
     def __init__(self, socket_name: str | None=None, allow_manual_search=False):
         if socket_name is None:
             socket_name = os.getenv("WAYFIRE_SOCKET")
-
+    
```

Comment:
Please don't add trailing whitespace. We should probably add a CI with a linter to catch these issues automatically.